experiments/modelfree/plot_noisy_observations.py

```diff
@@ -0,0 +1,141 @@
+import json
+import math
+import os
+
+from matplotlib import pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+RESULT_DIR = "/Users/cody/Data/adversarial_policies/ray_results/noisy_victim_obs/"
+OUT_DIR = "/Users/cody/Data/adversarial_policies/noisy_obs_plots/results_may_22"
+env_lookup = {
+    'SumoHumans': 'multicomp/SumoHumansAutoContact-v0',
+    'SumoAnts': 'multicomp/SumoAntsAutoContact-v0',
+    'KickAndDefend': 'multicomp/KickAndDefend-v0',
+    # 'YouShallNotPass': 'multicomp/YouShallNotPassHumans-v0' (Removed for now because I don't have
+    # time at the moment to fix the plotting logic to deal with victim in different location)
+}
+available_zoos = {
```

Comment:
Duplicate of `modelfree.envs.gym_compete.NUM_ZOO_POLICIES`

Reply:
Also, constants should be in all-caps: https://www.python.org/dev/peps/pep-0008/#constants